media-sound/fluid-soundfont/fluid-soundfont-3.1.ebuild

```diff
@@ -1,29 +1,62 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-DESCRIPTION="The Fluid R3 soundfont"
-HOMEPAGE="http://musescore.org/en/handbook/soundfont"
+# There are no new releases for old data files but the archival repo can update
+# the accompanying documentation [but this ebuild only cares for the data ;) ].
+COMMIT=700c7c1775da12f630750cc34149763ee52aa6c6
+
+DESCRIPTION="The Fluid R3 version 1 soundfont"
+HOMEPAGE="https://github.com/pinkflames/fluid-soundfont"
 SRC_URI="
-	http://prereleases.musescore.org/soundfont/${PN}_${PV}.tar.gz
-	timidity? ( https://dev.gentoo.org/~hwoarang/distfiles/timidity.cfg.bz2 )"
+	https://github.com/pinkflames/${PN}/archive/${COMMIT}.tar.gz -> ${PN}-${COMMIT}.tar.gz
+"
 
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64 ~ppc64 x86"
-IUSE="timidity"
+IUSE="+gm gs"
+
+BDEPEND="media-libs/flac"
+
+REQUIRED_USE="|| ( gm gs )"
+
+DOCS=( original/COPYING original/README )
+
+S="${WORKDIR}"/${PN}-${COMMIT}
+
+src_prepare() {
+	einfo "Using the flac tool to decompress the soundfont(s):"
+	use gm && flac --force-raw-format --endian=little --sign=signed -d original/FluidR3_GM.sf2.flacy -o FluidR3_GM.sf2
+	use gs && flac --force-raw-format --endian=little --sign=signed -d original/FluidR3_GS.sf2.flacy -o FluidR3_GS.sf2
 
-RDEPEND="timidity? ( app-eselect/eselect-timidity )"
+	default
```

Comment:
`default` should be the first thing called in `src_prepare`.

Reply:
My understanding was that calling it afterwards was a valid choice but sure; fixed.